app/components/support/user_search_result_component.html.erb

```diff
@@ -4,5 +4,8 @@
 
 <%= govuk_summary_list(classes: 'govuk-summary-list--no-border') do |component|
   component.slot(:row, key: 'Email address', value: user.email_address)
-  component.slot(:row, key: 'Associated with', value: unordered_list_of_orgs)
+  component.slot(:row,
+                  key: 'Associated with',
+                  value: unordered_list_of_orgs,
+                  action: (UserPolicy.new(current_user, user).editable? ? govuk_link_to('Change', associated_organisations_support_user_path(user)) : nil) )
```

Comment:
I **think** this can also be
```suggestion
                  action: (Pundit.policy(current_user, user).editable? ? govuk_link_to('Change', associated_organisations_support_user_path(user)) : nil) )
```
which may abstract the implementation detail of pundit a little bit more...